### DISCUSSION TITLE
what's the default pool about "device_health_metrics"? Can we disable to create it?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#9767] [Q&A] [answered]
@leseb

**Comment by leseb** [comment, marked as answer]
No, you cannot disable its creation, it is used as a common store for all `ceph-mgr` modules.

**Comment by zhucan** [comment]
if we can not disable  its creation, we want to create a new crush root with exists  osd,  it will cause https://github.com/rook/rook/discussions/10733

**Reply by travisn** [reply]
Try [this response](https://github.com/rook/rook/discussions/10733#discussioncomment-3425715).
